fog/distribution/src/main.rs

```diff
@@ -355,23 +355,28 @@ fn worker_thread_entry(
 
         // Send to the next fog account
         let to_account = &fog_accounts[txs_created % fog_accounts.len()];
+        // Sometime transactions could not be submitted before tombstone block passed,
+        // so loop until transactions can be submmitted
+        let mut done = false;
+        while !done {
```

Comment:
the code can be simpler if this is

```
loop {
...
```

and below, instead of `done = true;` we have `break`

ideally we would have a log message like "rebuilding failed tx" or something so that we know when we are going around this loop